packages/legend-shared/src/format/FormatterUtils.ts

```diff
@@ -56,6 +56,19 @@ export const TITLE_CASE_EXCEPTION_WORDS = [
   'up',
 ];
 
+export const prettyCONSTName = (value: string | undefined): string => {
+  if (!value) {
+    return '';
+  }
+  const containsId = value.slice(-2) === 'ID' || value.slice(-2) === 'Id';
```

Comment:
lets move `ID` and `Id` to const. 
lets also move out `value.slice(-2)` as like valueSuffix before comparing to make the code easier to read